templates/web/brent/report/new/roads_message.html

```diff
@@ -0,0 +1,4 @@
+    <div id="js-brent-tenant-services" class="hidden js-responsibility-message">
```

Comment:
I think I've done this right, but not working locally when the area selected. For a moment I thought it was because I hadn't added brent to `templates/web/fixmystreet.com/footer_extra_js.html` but I realise that isn't it as the assets have changes. It's probably just something I've done wrong, but can't see it

Reply:
That does look right, at least is same as other things that presumably work! What does happen, do you get an error, or just nothing happens at all? Is the asset data for Brent available in /js/asset_layers.js on your local fixmystreet cobrand? (Presumably is.)

Reply:
It just doesn't put the message up or block continuing. I don't see any errors - I'm happy to try it on staging as it's working for you and see how that works?

Reply:
weird. Yeah, sure! Here's what I get:
![image](https://user-images.githubusercontent.com/154364/203743514-5660fd87-244e-44e5-8909-446625e4b31e.png)

Only thing I can think of would be config difference, mine is:
```
    brent:
      - name: default
        http_wfs_url: "https://tilma.staging.mysociety.org/mapserver/brent"
        max_resolution: 4.777314267158508
        geometryName: 'msGeometry'
        srsName: "EPSG:3857"
        body: "Brent Council"
      - asset_type: 'area'
        stylemap: fixmystreet.assets.stylemap_invisible
        nearest_radius: 0.1
        always_visible: true
        non_interactive: true
        all_categories: true
        no_asset_msg_id: '#js-brent-tenant-services'
        actions:
          found: fixmystreet.assets.brent.found
          not_found: fixmystreet.assets.brent.not_found
        road: true
        wfs_feature: "Housing"
```

Reply:
This was mine, but just copied yours and still getting the same issue
```    brent:
      - name: default
        http_wfs_url: "https://tilma.staging.mysociety.org/mapserver/brent"
        max_resolution: 4.777314267158508
        geometryName: 'msGeometry'
        srsName: "EPSG:3857"
        body: "Brent Council"
      - asset_type: 'area'
        stylemap: fixmystreet.assets.stylemap_invisible
        non_interactive: true
        nearest_radius: 0.1
        always_visible: true
        all_categories: true
        no_asset_msg_id: '#js-brent-tenant-services'
        actions:
          found: fixmystreet.assets.brent.found
          not_found: fixmystreet.assets.brent.not_found
        road: true
        wfs_feature: "Housing"